src/custom/state/orders/updaters/PendingOrdersUpdater.ts

```diff
@@ -13,6 +19,24 @@ import { OrderID } from 'api/gnosisProtocol'
 
 import { fetchOrderPopupData, OrderLogPopupMixData } from 'state/orders/updaters/utils'
 
+/**
+ * Return the ids od the orders that we are not yet aware that are signed.
+ * This is, pre-sign orders, in state of "PRESIGNATURE_PENDING", for which we now know they are signed
+ *
+ * Used as an auxiliar method to detect which orders we should mark as pre-signed, so we change their state
+ *
+ * @param allPendingOrders All pending orders
+ * @param signedOrdersIds ids of orders we know are already pre-signed
+ * @returns ids of the pending orders that were pending for pre-sign, and we now know are pre-signed
+ */
+function _getPresignedOrdersStillWaiting(allPendingOrders: Order[], signedOrdersIds: OrderID[]) {
```

Comment:
The fn name is quite confusing.
I spent a few minutes staring at it because it says something and does something else.

If I understand what this is doing, the name should be something like: `getNewlyPreSignedOrders`

Reply:
That's a better name!